app/services/api/v3/countries_wb_indicators/api_service.rb

```diff
@@ -0,0 +1,65 @@
+require 'net/http'
+
+module Api
+  module V3
+    module CountriesWbIndicators
+      class ApiService
+        INDICATORS = {
+          population: 'SP.POP.TOTL',
+          gdp: 'NY.GDP.MKTP.CD',
+          land_area: 'ag.lnd.totl.k2',
+          agricultural_land_area: 'AG.LND.AGRI.K2',
+          forested_land_area: 'AG.LND.FRST.K2'
+        }.freeze
+
+        def self.population_indicators(iso_code = 'ALL')
+          indicator_request(:population, iso_code)
+        end
+
+        def self.gdp_indicators(iso_code = 'ALL')
+          indicator_request(:gdp, iso_code)
+        end
+
+        def self.land_area_indicators(iso_code = 'ALL')
+          indicator_request(:land_area, iso_code)
+        end
+
+        def self.agricultural_land_area_indicators(iso_code = 'ALL')
+          indicator_request(:agricultural_land_area, iso_code)
+        end
+
+        def self.forested_land_area_indicators(iso_code = 'ALL')
+          indicator_request(:forested_land_area, iso_code)
+        end
+
+        private_class_method def self.indicator_request(name, iso_code)
+          uri = URI(
+            "#{ENV['WORLD_BANK_API_URL']}/v2/country/#{iso_code}/indicator/" \
+            "#{INDICATORS[name]}?format=json&per_page=10000"
```

Comment:
is it possible to filter by year range? I think WB has ~50 years of data, we need much less in Trase, so that could be a way to optimise